### DISCUSSION TITLE
language_model/mmlm/mmlm_intro/

### DISCUSSION THREAD

**Discussion by giscus** [ritchieng/deep-learning-wizard#26] [Website]
# language_model/mmlm/mmlm_intro/

We try to make learning deep learning, deep bayesian learning, and deep reinforcement learning math and code easier. Open-source and used by thousands globally.

https://www.deeplearningwizard.com/language_model/mmlm/mmlm_intro/

**Comment by AnandSiyote** [comment]
response = ollama.chat(
    model="llava:13b-v1.6",
    messages=[message]
)

For this to run in local jupyter file : 
1.Download Ollama exe, run it on your machine.
2. Start Ollama app
3. Open Terminal and run commands : 
ollama serve
ollama pull llava:13b-v1.6

**Reply by ritchieng** [reply]
Thanks for this! Jupyter notebook is available for download in the [Github repo](https://github.com/ritchieng/deep-learning-wizard/blob/master/docs/language_model/mmlm/mmlm_intro.ipynb) too!
